libraries/Wire/Wire.h

```diff
@@ -39,6 +39,7 @@ class TwoWire : public Stream
     void begin(uint8_t);
     void end();
     void setClock(uint32_t);
+	void setPins(uint8_t pin_SDA, uint8_t pin_SCL);
```

Comment:
please fix the indent (better to use spaces over tab), also the variable name should be pinSDA/pinSCL to be consistent with constructor.